index.html

```diff
@@ -0,0 +1,221 @@
+            </p>
+         <div class="headline-social">
+            <h3 class="social-title">LET'S CONNECT</h3>
+            <ul class="social-links">
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/twitter.svg" alt="Twiter logo"/></a>
+                </li>
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/Linkedin icon.svg" alt="Linkdin logo"/></a>
+                </li>
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/medium.svg" alt="Medium logo" /></a>
+                </li>
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/github.svg" alt="Github Logo" /></a>
+                </li>
+                <li class="social-link">
+                    <a href="#"><img class="social-icon" src="/images/angellist.svg" alt="Angellist" /></a>
+                </li>
+            </ul>
+         </div>
+        </section>
+    </main>
+   </section>
+   <section class="work-section">
+    <div class="work-border">
+        <img class="portfolio-img" src="/images/Snapshoot Portfolio.png" alt="portfolio image" />
+    <div class="portfolio-section">
+        <h3 class="project-name">Tonic</h3>
+    <div class="project-details">
+        <h4 class="project-owner">CANOPY</h4>
+        <img class="cunter" src="/images/Counter.png" alt="counter" />
+        <p class="play-role">Back End Dev</p>
+        <img class="cunter" src="/images/Counter.png" alt="counter" />
+        <p class="year">2015</p>
+    </div>
+     <p class="project-description">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+     </p>
+     <ul class="p-languages">
+        <li class="language">html</li>
+        <li class="language">css</li>
+        <li class="language">java script</li>
+     </ul>
+     <button class="button" type="button">See Project</button>
+     </div>
+    </div>
+
+   <div class="work-border-two">
+        <img class="portfolio-img" src="/images/Snapshoot Portfolio2.png" alt="portfolio image" />
+    <div class="portfolio-section">
+        <h3 class="project-name">Multi-Post Stories</h3>
+    <div class="project-details">
+        <h4 class="project-owner">CANOPY</h4>
+        <img class="cunter" src="/images/Counter.png" alt="counter" />
+        <p class="play-role">Back End Dev</p>
+        <img class="cunter" src="/images/Counter.png" alt="counter" />
+        <p class="year">2015</p>
+    </div>
+     <p class="project-description">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+     </p>
+     <ul class="p-languages">
+        <li class="language">html</li>
+        <li class="language">css</li>
+        <li class="language">java script</li>
+     </ul>
```

Comment:
- [ ] Kindly use capital letters to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/ariful04/Myportfolio/assets/58642949/3d4d8bac-0296-45b6-82a6-5cd3de741295) | ![image](https://github.com/ariful04/Myportfolio/assets/58642949/35eca267-9978-4503-90cb-6597602408c1) |